test/db/formats/pe/sectiondata

```diff
@@ -28,7 +28,7 @@ NAME=PE: corkami cm01.exe - section seek 1
 FILE=bins/pe/cm01.exe
 CMDS=<<EOF
 s 0x00401000
-om.~[10]
+om.~[8]
```

Comment:
i think it will be good to have a test covering the actual output without any grep, so we can confirm the output doesnt changes and behaves properly

Reply:
yeah, but that is out of scope of this pr imo

Reply:
Having the full output of om and om. in a single test is useful and doesnt cost more than 4 lines, 3 of them copypasted